lib/AACStream/aacs.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * mux.js
+ *
+ * Copyright (c) 2016 Brightcove
+ * All rights reserved.
+ *
+ * A stream-based aac to mp4 converter. This utility can be used to
+ * deliver mp4s to a SourceBuffer on platforms that support native
+ * Media Source Extensions.
+ */
+'use strict';
+var Stream = require('../utils/stream.js');
+
+// object types
+var AacStream;
+
+var
```

Comment:
Because of the way `var` is being used here, you are defining AacStream twice. I suspect one is just leftover.